index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@400&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header id="sub-container">
+      <nav class="navbar">
+        <a href="#" class="logo">My Logo</a>
+        <a href="#" class="menu">
+          <img src="./images/menu/Icon-Menu.png" alt="menu icon" />
+        </a>
+      </nav>
+    </header>
+    <section class="main-container">
+      <h1 class="greeting">
+        Hey there. <br />
+        I'm John
+      </h1>
+      <div class="sub-main-container">
+        <section>
+          <div class="intro">
+            <h3 class="profession">A Web Developer</h3>
+            <p class="welcome">
+              I can help you build a product , feature or <br />
+              website Look through some of my work <br />
+              and experience! If you like what you see <br />
+              and have a project you need coded,
+            </p>
+          </div>
+        </section>
+        <div class="design"><img src="./images/design.png" alt="design" /></div>
+        <div class="connect-text">Don't hestiate to contact me.</div>
+        <div class="connect-link">
+          <ul>
+            <li>
+              <a href="https://github.com/Outkarm"
+                ><img
+                  src="./images/github-logo/github.png"
+                  alt="github-logo"
+                  class="github"
+              /></a>
+            </li>
+            <li>
+              <a href="https://www.linkedin.com/in/john-kpordje-866749241/"
+                ><img
+                  src="./images/linked-in-logo/linked-in-2.png"
+                  alt="linked-in-logo"
+                  class="linked-in"
+              /></a>
+            </li>
+            <li>
+              <a href="https://twitter.com/outkarm"
+                ><img
+                  src="./images/twitter-logo/twitter.png"
+                  alt="twitter"
+                  class="twitter"
+              /></a>
+            </li>
```

Comment:
- [ ] [Optional] This is an optional suggestion, Kindly implement it, you have only three social media icons please try to use the remaining 2 icons as well. You can see the design below for reference.
### Your design
![your](https://user-images.githubusercontent.com/81727988/185508890-563b35ed-a878-447e-aa62-c5851e5f7faf.png)
### Figma design
![hhhhhhhh](https://user-images.githubusercontent.com/81727988/185508910-2bda93b6-71af-460c-b06f-c4726fc630d7.png)